test_plans/kernel.asciidoc

```diff
@@ -0,0 +1,39 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl::kernel (cover gaps)
+
+This is a test plan cover sycl::kernel functionality that is not covered by current tests.
+
+== Testing scope
+
+Tests intend to cover functions from 4.11.13. The kernel class that are not covered in https://github.com/KhronosGroup/SYCL-CTS/tree/SYCL-2020/tests/kernel
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+== Tests
+
+=== Deleted default constructor
+
+Check that std::is_default_constructible_v<sycl::kernel> is `false`.
+
+=== Test for `get_backend()`

```

Comment:
In all of the following tests it's not clear how you will get the `kernel` object.  You could add some common description that applies to all of the tests like:

> * Define a simple kernel named `KernelName` that uses no optional features.
> * Obtain a kernel bundle in `executable` state that contains this kernel via `get_kernel_bundle<KernelName, bundle_state::executabe(const context & const device &)`.
> * Get a `kernel` object for the kernel by calling `kernel_bundle::get_kernel<KernelName>()`.

Reply:
Thank you, clarification how to obtain object is added.